sorc/chgres_cube.fd/search_util.f90

```diff
@@ -149,22 +166,39 @@ subroutine search (field, mask, idim, jdim, tile, field_num, latitude)
        elseif (field_num == 91) then  ! sea ice fract
          if (abs(latitude(i,j)) > 55.0) then
            field(i,j) = default_value
+           repl_default = repl_default + 1
          else
            field(i,j) = 0.0
+           repl_default = repl_default + 1
          endif
+       elseif (field_num == 7) then 
```

Comment:
Why does chgres need to adjust the terrain?  The 'oro' files provide the 'official' land mask and terrain.  chgres should map to it.

Reply:
This only happens on the terrain_from_input_grid field. Despite the indication in this file that a flag value of  -99999.9 in the terrain height will turn off soil temperature adjustments for terrain height, that is not implemented in adjust_soilt_for_terrain. So, when using some of the new fields from HRRR data, I was getting isolated flag values at land ice points in input terrain height after interpolating it to the target grid, thus the soil temperature adjustment would fail. If we'd like to implement that flag value to turn off soil temperature adjustment, that would work instead.

Reply:
@GeorgeGayno-NOAA I've updated the PR to include code changes from the other comments in this thread.  If we want to turn off soil temperature adjustment over these isolated points, that would be the last things to change.  Thanks.